### DISCUSSION TITLE
Risk score

### DISCUSSION THREAD

**Discussion by sbpoc2022** [DependencyTrack/dependency-track#1824] [Q&A]
How should the consumer interpret risk score coming from dependency tracker? Is there a guideline to see risk score in the context of some other attribute or is there a t-shirt size qualitative rating that maps to some range? I couldn't find much reference. Any help?

**Comment by sbpoc2022** [comment]
Found my answer here...
https://github.com/DependencyTrack/dependency-track/discussions/894

**Comment by msymons** [comment]
@sbpoc2022, also have a look at EPSS in Dependency-Track for a measure of risk.
